packages/flutter/test/material/date_range_picker_test.dart

```diff
@@ -268,6 +268,15 @@ void main() {
     });
   });
 
+  testWidgets('Long helpText cant not cutoffs the save button', (WidgetTester tester) async {
```

Comment:
'Long helpText does not cutoff the save button'

Reply:
@justinmc Thanks for your review, I have updated the test.